provisioning/device/src/interfaces.ts

```diff
@@ -140,10 +134,9 @@ export interface TpmRegistrationResult extends RegistrationResult {
 /**
  * @private
  */
-export interface TpmProvisioningTransport {
+export interface TpmProvisioningTransport extends PollingTransportHandlers {
+  setSasToken(sasToken: string): void;
```

Comment:
>setSasToken [](start = 2, length = 11)

I think this would be the ek/srk pair instead - i'd advocate to remove that until we actually tackle TPM implementation